frontend/public/components/machine-config-pool.tsx

```diff
@@ -246,12 +246,12 @@ export const MachineConfigPoolDetailsPage: React.SFC<any> = (props) => (
 );
 
 const tableColumnClasses = [
-  'pf-u-w-25-on-lg pf-u-w-33-on-md pf-u-w-50-on-sm',
-  'pf-m-hidden pf-m-visible-on-xl pf-u-w-50-on-xl',
-  'pf-u-w-25-on-sm pf-u-w-16-on-md pf-u-w-8-on-lg',
-  'pf-m-hidden pf-m-visible-on-md pf-u-w-16-on-md pf-u-w-8-on-lg',
-  'pf-m-hidden pf-m-visible-on-lg pf-u-w-8-on-lg',
-  'pf-u-w-25-on-lg pf-u-w-33-on-sm pf-u-w-8-on-lg',
+  'pf-u-w-33-on-md pf-u-w-25-on-lg',
```

Comment:
It seems like other instances the sizes descend from largest to smallest.  Would be good to be consistent.

Reply:
I updated the sizes to consistent. Going from sm > md > lg > xl >2xl since PF rules apply to `*-on-{size}` and larger